pkg/issuer/acme/setup.go

```diff
@@ -97,6 +97,9 @@ func (a *Acme) Setup(ctx context.Context) error {
 		// We clear the ACME account URI as we have generated a new private key
 		a.issuer.GetStatus().ACMEStatus().URI = ""
 
+	case a.issuer.GetSpec().ACME.OnlyUseExistingAccountKey && apierrors.IsNotFound(err):
+		return fmt.Errorf("the ACME issuer config has 'onlyUseExistingAccountKey' set to true, but the secret was not found: %w", err)
```

Comment:
Does this need to set a condition on the issuer to indicate that it isn't ready.

Reply:
I'm not sure if it *needs* to be set, but it sounds like a good idea.